src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -254,11 +254,9 @@ def ls_factor(stream_length, area, avg_slope, m):
         return ls
 
 
-def stream_length(geom, drb=False):
```

Comment:
Yeah, looks like just two calls to `stream_length` in `tasks.py`:

This one sets the value for `z['StreamLength']`: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/tasks.py#L123

And this one's in the DRB logic I cut: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/tasks.py#L181

Here's the call to `streams`, which has a similar signature: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/tasks.py#L475